core/managers.py

```diff
@@ -41,6 +41,15 @@ def agents_only(self):
     def structures_only(self):
         return self.exclude(structure__isnull=True)
 
+    def with_active_user(self):
+        return self.filter(Q(structure__isnull=False) | Q(agent__user__is_active=True))
```

Comment:
car intégré dans la suggestion de `can_be_emailed` et que structure n'a pas de lien avec user 
```suggestion
        return self.filter(Q(agent__user__is_active=True))
```